File: client/src/app/shared/models/birthdays/serviceincluded.ts
export interface ServiceIncluded {
    id: number;
    name: string;
    description: string;
    picture: string;
    videoClip: string;
}

export interface ServiceIncludedCreateEdit {
    id: number;
    name: string;
    description: string;
    picture: File;
    videoClip: string;
}

File: Core/Entities/Birthday/ServiceIncluded.cs
using System.Collections;
using System.Collections.Generic;

namespace Core.Entities
{
    // ovo si stavio u birthdy file, u stvarnosti to stavi u neki nadfajl
    public class ServiceIncluded : BaseEntity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Picture { get; set; }
        public string VideoClip { get; set; }

        public ICollection<BirthdayPackageService> BirthdayPackageServices { get; set; }
    }
}